src/itdelatrisu/opsu/GameData.java

```diff
@@ -756,8 +756,10 @@ else if (health.getHealth() >= 25f)
 			// combo burst
 			if (comboBurstIndex != -1 && comboBurstAlpha > 0f) {
 				Image comboBurst = comboBurstImages[comboBurstIndex];
+				//TODO flip image if drawn on the right side of the screen
+				//if (comboBurstX <= width - comboBurst.getWidth()) comboBurst = comboBurst.getFlippedCopy(true, false);
```

Comment:
[The image flips **after** its done moving to the side.](https://gfycat.com/BossyLiveGharial)
**Removed** for "cleaner" code